### DISCUSSION TITLE
[QUESTION] Fullscreen option still present?

### DISCUSSION THREAD

**Discussion by arcolinuxz** [davatorium/rofi#1482] [Theming]
### Rofi version (rofi -v)

Version : 1.7.0-1

### Configuration

https://gist.github.com/arcolinuxz/4f1d62ae5be8c11702b6c00611defece

### Launch command

i3 : bindsym $mod+F11 exec --no-startup-id rofi -show drun -fullscreen -font "Noto Sans 13"

### Step to reproduce

This code

`bindsym $mod+F12 exec --no-startup-id rofi -show drun -font "Noto Sans 13"`

looks exactly the same as this code

`bindsym $mod+F11 exec --no-startup-id rofi -show drun -fullscreen -font "Noto Sans 13"`

and I wonder why.

Running this on ArcoLinux - based on Arch Linux

### Expected behavior

full screen

### Actual behavior

not full screen

**Comment by TonCherAmi** [comment]
The [release notes for 1.7.0](https://github.com/davatorium/rofi/releases/tag/1.7.0) clearly state that `-fullscreen` has been removed.

**Reply by azm6** [reply]
You can still do this `-theme-str 'window {width: 100%;height: 100%;}'`

**Comment by DaveDavenport** [comment]
```css
window { 
    fullscreen: true;
}
````
